src/sass/components/_table.scss

```diff
@@ -0,0 +1,38 @@
+.table-wrapper {
+	overflow-x: auto;
+	width: 100%;
+	margin: 0 auto;
+}
+
+.c-table {
+	position: relative;
+	font-size: 1.8rem;
+	font-weight: 500;
+	color: $white;
+	border-collapse: collapse;
+	padding: 5rem;
+	width: 90rem;
+	margin: 2rem auto;
+
+}
+
+td, th {
+	padding: 1rem 5rem;
+	text-align: left;
+}
```

Comment:
We should scope this. Add some class to those elements or nest under parent selector.